### PR TITLE
Make build compatible with new MSBuild alpha

### PR DESCRIPTION
**Pull request by aensidhe**

### CODE REVIEW

**Review by westsouthnight — APPROVED** [review]
Its no bad idea

File: src/msgpack.light/msgpack.light.csproj [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" />
+    <EmbeddedResource Include="**\*.resx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NETStandard.Library" Version="1.6" />
+    <PackageReference Include="JetBrains.Annotations">
```
Comment (aensidhe) [inline src/msgpack.light/msgpack.light.csproj]: Let's unify xml stuff here

File: tests/msgpack.light.tests/msgpack.light.tests.csproj [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <Compile Include="**\*.cs" />
+    <EmbeddedResource Include="**\*.resx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20161123-03" />
+    <PackageReference Include="xunit" Version="2.2.0-beta4-build3444" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0-beta4-build1194" />
+    <PackageReference Include="JetBrains.Annotations">
```
Comment (aensidhe) [inline tests/msgpack.light.tests/msgpack.light.tests.csproj]: And here.

Look here: https://blogs.msdn.microsoft.com/dotnet/2016/12/12/updating-visual-studio-2017-rc-net-core-tooling-improvements/

**Review by aensidhe — COMMENTED** [review]
Good

**Review by roman-kozachenko — APPROVED** [review]

**Pull request merged**